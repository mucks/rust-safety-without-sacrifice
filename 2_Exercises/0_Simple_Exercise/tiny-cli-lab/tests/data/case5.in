# Random mix
REV The quick brown fox
ADD 42 58
COUNT programming
UPPER already UPPER?
MUL 3 7
