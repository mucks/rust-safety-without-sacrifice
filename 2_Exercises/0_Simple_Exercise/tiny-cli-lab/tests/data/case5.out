fox brown quick The
100
11
ALREADY UPPER?
21
